iodata/test/test_mwfn.py

```diff
@@ -0,0 +1,210 @@
+    assert_equal(mol.extra['mo_sym'][0], '?')
+    # test that for the same molecule fchk and mwfn generate the same objects.
+    olp = compute_overlap(mol.obasis, mol.atcoords)
+    mol2 = load_fchk_helper('ch3_rohf_sto3g_g03.fchk')
+    olp_fchk = compute_overlap(mol2.obasis, mol2.atcoords)
+    # assert_allclose fails due to abs/rel tolerance settings
+    assert np.allclose(mol.atcoords, mol2.atcoords)
+    assert_allclose(mol2.obasis.shells[0].coeffs, coeffs1)
+    # fails due to special SP basis storage (leaving commented lines in for future reference)
+    # assert_allclose(mol2.obasis.shells[1].coeffs, coeffs2)
+    # assert_allclose(mol2.obasis.shells[2].coeffs, coeffs3)
+    assert_allclose(mol2.obasis.shells[3].coeffs, coeffs4)
+    assert_allclose(mol2.obasis.shells[4].coeffs, coeffs4)
+    # assert_allclose fails due to abs/rel tolerance settings
+    assert np.allclose(olp, olp_fchk)
+
+
+def test_load_mwfn_ch3_hf_g03():
+    mol = load_mwfn_helper('ch3_hf_sto3g_fchk_multiwfn3.7.mwfn')
+    assert_equal(mol.mo.occs.shape[0], mol.mo.coeffs.shape[1])
+    assert_equal(mol.extra['wfntype'], 1)
+    # test first molecular orbital information
+    coeff = np.array([9.91912304E-01, 3.68365244E-02, 9.23239012E-04, 9.05953703E-04,
+                      9.05953703E-04, -7.36810756E-03, - 7.36810756E-03, - 7.36919429E-03])
+    assert_equal(mol.mo.coeffs[:, 0], coeff)
+    mo_energies = np.array([-1.10094534E+01, -9.07622407E-01, -5.37709620E-01, -5.37273275E-01,
+                            -3.63936540E-01, 6.48361367E-01, 7.58140704E-01, 7.59223157E-01,
+                            -1.09780991E+01, -8.01569083E-01, -5.19454722E-01, -5.18988806E-01,
+                            3.28562907E-01, 7.04456296E-01, 7.88139770E-01, 7.89228899E-01])
+    assert_allclose(mol.mo.energies, mo_energies)
+    assert_equal(mol.mo.occs[0], 1.000000)
+    assert_equal(mol.extra['mo_type'][0], 1)
+    assert_equal(mol.extra['mo_sym'][0], '?')
+    # test that for the same molecule fchk and mwfn generate the same objects.
+    olp = compute_overlap(mol.obasis, mol.atcoords)
+    mol2 = load_fchk_helper('ch3_hf_sto3g.fchk')
+    olp_fchk = compute_overlap(mol2.obasis, mol2.atcoords)
+    # assert_allclose fails due to abs/rel tolerance settings
+    assert np.allclose(mol.atcoords, mol2.atcoords)
+    # assert_allclose fails due to abs/rel tolerance settings
+    assert np.allclose(olp, olp_fchk)
+
+
+def test_nelec_charge():
+    mol1 = load_mwfn_helper('ch3_rohf_sto3g_g03_fchk_multiwfn3.7.mwfn')
+    assert mol1.nelec == 9
+    assert mol1.charge == 0
+    mol2 = load_fchk_helper('he_spdfgh_virtual_fchk_multiwfn3.7.mwfn')
+    assert mol2.nelec == 2
+    assert mol2.charge == 0
+    assert mol2.extra['nelec_a'] == 1
+    assert mol2.extra['nelec_b'] == 1
+    mol3 = load_fchk_helper('ch3_hf_sto3g_fchk_multiwfn3.7.mwfn')
+    assert mol3.nelec == 9
+    assert mol3.charge == 0
+    assert mol3.extra['nelec_a'] == 5
+    assert mol3.extra['nelec_b'] == 4
```

Comment:
These may be affected by #196

Reply:
I just realized that mol2 and mol3 use the load_fchk_helper rather than the load_mwfn_helper. Obviously, one of the helpers is superfluous since they use load_one from the api, which bases its decisions on the format. I will remove one of the two and give the helper function a generic name.